transmogrifier/sources/ead.py

```diff
@@ -117,6 +118,86 @@ def get_optional_fields(self, xml: Tag) -> Optional[dict]:
 
         # format field not used in EAD
 
+        # holdings
+        for holding_element in collection_description.find_all(
+            "originalsloc", recursive=False
+        ):
+            if holding_value := self.create_string_from_mixed_value(
+                holding_element, " ", ["head"]
+            ):
+                fields.setdefault("holdings", []).append(
+                    timdex.Holding(note=holding_value)
+                )
+        for holding_element in collection_description_did.find_all(
```

Comment:
Do you think we should be creating a single `Holding` with `note=originalsloc` and `location=physloc` (if those value exist)?

Or should there be conditional block like this:

```
if collection_description.find("originalsloc"):
  ...create Holding
elif collection_description_did.find("originalsloc"):
  ...create Holding
```

Re: multiple `phyloc`'s, we've generally been treating instances of a field as separate entities, aside from when we're forced to concatenate into a string (the upcoming `physical_description`). But if it makes sense to dump them all in a string for a single `Holding` record, I can do that.

Reply:
Well, the question for this field is: conceptually, when we encounter both an `originalsloc` and a `physloc` field in a record, do they represent the same holding or different holdings? Same question with multiple `physloc` fields in a record. And if they are different holdings, how do we know and how is that represented in the data? My sense about archival collections is that there is really only one "holding" per collection, because the items are unique. That may not be true for originals vs duplicates or physical vs digital versions, but if the data doesn't clearly indicate why they are showing up as separate holdings then we may be causing confusion. This might be an area where we should discuss it further with the archives staff to better understand how they conceptualize it.

Reply:
I'll get a slack chat or email going on this with us and some DDC staff to get their perspective

Reply:
Yes, that chat happened after I submitted this commit, I already have it in place for the next commit

Reply:
Is there any particular reason not to push that change as new commit for this PR? I think 1) the holdings field conversation and decision to omit it for now are part of this PR so should be included in it, and 2) I would prefer not to approve a PR that contains code we intend to immediately delete.